### PR TITLE
added strategy docs link

### PR DESCRIPTION
**Pull request by smartdevcom**

### CODE REVIEW

File: packages/chains/src/bsc/plugins.ts [hunk 1]
```diff
@@ -4,32 +4,38 @@ const deployedPlugins: DeployedPlugins = {
...
+    strategyDocsUrl:
+      "https://app.gitbook.com/s/66NW0CnbA4sgMbZnOH21/guides/assets-and-strategies-addresses/binance-smart-chain-bsc/bomb",
   },
   "0x3c29e9b0CfE6FfF97f373eAbEADE9475FaC3bd4e": {
     market: "0xf0a2852958aD041a9Fb35c312605482Ca3Ec17ba",
     name: "2brl DotDotLpERC4626",
-    docsUrl: "https://dotdot.finance/#/stake",
+    apyDocsUrl: "https://dotdot.finance/#/stake",
+    strategyDocsUrl:
+      "https://app.gitbook.com/s/66NW0CnbA4sgMbZnOH21/guides/assets-and-strategies-addresses/binance-smart-chain-bsc/jarvis-jfiat-pool",
```
Comment (carlomazzaferro) [inline packages/chains/src/bsc/plugins.ts:23]: These are the private gitbook address and won't work for anyone else

File: packages/chains/src/bsc/plugins.ts [hunk 2]
```diff
@@ -4,32 +4,38 @@ const deployedPlugins: DeployedPlugins = {
...
   },
   "0x9dB349BbfF9E177dB4bd3134ff93876688b77835": {
     market: "0xccc9BEF35C50A3545e01Ef72Cc957E0aec8B2e7C",
     name: "val3EPS DotDotLpERC4626",
-    docsUrl: "https://dotdot.finance/#/stake",
+    apyDocsUrl: "https://dotdot.finance/#/stake",
```
Comment (carlomazzaferro) [inline packages/chains/src/bsc/plugins.ts:28]: Why do these not have `strategyDocsUrl`?

File: packages/chains/src/polygon/plugins.ts [hunk 3]
```diff
@@ -4,37 +4,51 @@ const deployedPlugins: DeployedPlugins = {
   "0x6578e774120F6010315784C69C634bF3946AFb0c": {
     market: "0x9b5D86F4e7A45f4b458A2B673B4A3b43D15428A7",
     name: "Beefy agEUR-jEUR Vault",
-    docsUrl: "https://app.beefy.com/vault/jarvis-2eur",
+    apyDocsUrl: "https://app.beefy.com/vault/jarvis-2eur",
+    strategyDocsUrl:
+      "https://app.gitbook.com/s/66NW0CnbA4sgMbZnOH21/guides/assets-and-strategies-addresses/polygon/jarvis-jfiat-pool",
```
Comment (carlomazzaferro) [inline packages/chains/src/polygon/plugins.ts:9]: same for all of these

**Review by carlomazzaferro — CHANGES_REQUESTED** [review]
see comments

File: packages/chains/src/bsc/plugins.ts (hunk 2 above)
Comment (smartdevcom) [reply packages/chains/src/bsc/plugins.ts:28]: I thought these were Ellipsis pool and docs are provided for `Jarvis` and `Bomb` pool only

File: packages/chains/src/bsc/plugins.ts (hunk 1 above)
Comment (smartdevcom) [reply packages/chains/src/bsc/plugins.ts:23]: Should other url be used for docs link?

Comment (carlomazzaferro) [reply packages/chains/src/bsc/plugins.ts:23]: Yes of course, they should be the docs.midascapital.xyz , not the private gitbook addresses

File: packages/chains/src/bsc/plugins.ts (hunk 2 above)
Comment (carlomazzaferro) [reply packages/chains/src/bsc/plugins.ts:28]: Any asset that uses erc4636 strategies should have a strategyUrl

File: packages/chains/src/bsc/plugins.ts (hunk 1 above)
Comment (smartdevcom) [reply packages/chains/src/bsc/plugins.ts]: Added public docs urls

File: packages/chains/src/bsc/plugins.ts (hunk 2 above)
Comment (smartdevcom) [reply packages/chains/src/bsc/plugins.ts:28]: @NikMidas will add some docs missing now. I guess we are missing `Arrakis` docs for Polygon and `Ellipsis` docs for BSC.

**Review by carlomazzaferro — APPROVED** [review]

**Comment by smartdevcom** [comment]
I will merge this PR after adding the docs url which @NikMidas is working on

**Pull request merged**